src/azul/service/file_service.py

```diff
@@ -0,0 +1,102 @@
+import json
+from typing import (
+    Mapping,
+)
+
+from elasticsearch_dsl.response import (
+    Hit,
+)
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    CatalogName,
+    cached_property,
+    config,
+)
+from azul.drs import (
+    DRSClient,
+)
+from azul.plugins import (
+    RepositoryPlugin,
+)
+from azul.service import (
+    Filters,
+)
+from azul.service.elasticsearch_service import (
+    ElasticsearchService,
+)
+from azul.tdr import (
+    TDRClient,
+)
+from azul.types import (
+    JSON,
+)
+
+
+class FileService(ElasticsearchService):
```

Comment:
```suggestion
class RepositoryService(ElasticsearchService):
```

and RepositoryService -> IndexService

Discuss at stand-up. Quick, separate PR?